src/main/groovy/com/cedarsoftware/ncube/NCubeManager.groovy

```
@@ -2533,7 +2533,11 @@ target axis: ${transformApp} / ${transformVersion} / ${transformCubeName}, user:
         {
             Connection connection = threadBoundConnection
             connection.rollback() // rollback in case any work was done before this point
-            obsoletePullRequest(prId)
+            try
+            {
+                obsoletePullRequest(prId)
+            }
+            catch (IllegalArgumentException ignore) {} // pr was rolled back
```

This should be logged.  Eating exceptions is generally a bad practice. If it is logged, then that mitigates hiding the exception.